internal/controller/preservation/preservation.go

```diff
@@ -254,5 +202,43 @@ func Cleanup(
 		}
 	}
 
-	return completedCleanup, errors.NewAggregate(errs)
+	return completedCleanup, utilerrors.NewAggregate(errs)
+}
+
+func GetPreservedResourceCounts(
```

Comment:
Where are these used and what do they have to do with preserving resources that are internal to the cluster instance?

Reply:
`GetPreservedResourceCounts` and `GetRestoredResourceCounts` will be used by a Reinstall Handler in SiteConfig (subsequent PR) to report on the status of the number of resources preserved / restored. The reason these are here is to make use of the label selectors without having to expose them. Furthermore, these functions are better suited in the `preservation` package as they process backup & restore data.

Reply:
As discussed on slack, I will isolate the `GetResourceCounts` functions and unit tests in a separate PR.

Reply:
PR to be posted once this PR is merged due to dependencies